RUFAS/routines/manure/beddings/bedding_classes.py

```diff
@@ -11,153 +11,254 @@
 
 
 class BeddingType(DefaultEnum):
-    """Enumerates the different types of bedding."""
+    """
+    A class used to enumerate the different types of bedding.
+
+    This class is derived from the `DefaultEnum` base class. It provides a set of predefined constants
+    that represent different types of bedding, such as sawdust, straw, and sand. The default type is sand.
```

Comment:
```suggestion
    Provides a set of predefined bedding types, such as sawdust, straw, and sand.
    
    The default type is sand
 
```
Do you think a more concise description is better here?

Reply:
I simplified the summary line a bit. Also, I believe the current convention is that the starting verb in a docstring should be in the imperative form, though I still have inconsistencies with this here and there.